packages/grafana-data/src/types/data.ts

```diff
@@ -76,7 +76,6 @@ export interface QueryResultMeta {
   /**
    * Legacy data source specific, should be moved to custom
    * */
-  alignmentPeriod?: number; // used by cloud monitoring
```

Comment:
This is left-over from [this migration](https://github.com/grafana/grafana/pull/38650/files#diff-be823fa860127465e4c9518eab333dec6bbac9b158c01a59f62ac302cd825269L47). It's safe to remove.